### ISSUE TITLE
Trigger a compliance API check when you app deploys

### ISSUE THREAD

**Issue by maxneuvians** [cds-snc/report-a-cybercrime#125]
The intent is to trigger a compliance API check whenever you app has deployed and ready. Because this is a non-deterministic event, the requirement is for the app to decide when the best point in time for that is.

The compliance watcher application exposes an internal kubernetes endpoint (https://github.com/cds-snc/report-a-cybercrime/blob/master/compliance/watcher-deployment.yaml#L34), which can be triggered by a POST request.

In other words, you need to make a POST request to:

`compliance-watcher.symmorfosi.svc.cluster.local:3000/run_compliance`

to trigger a compliance run. I would recommend putting it in here or thereabouts:

https://github.com/cds-snc/report-a-cybercrime/blob/master/frontend/src/index.js#L22
